app/src/main/java/com/duckduckgo/app/email/ui/EmailLogoutDialog.kt

```diff
@@ -18,23 +18,17 @@ package com.duckduckgo.app.email.ui
 
 import android.app.Dialog
 import android.os.Bundle
-import android.view.View
 import androidx.appcompat.app.AlertDialog
 import androidx.fragment.app.DialogFragment
 import com.duckduckgo.app.browser.R
-import com.google.android.material.textview.MaterialTextView
 
 class EmailLogoutDialog : DialogFragment() {
 
     var onLogout: (() -> Unit) = {}
 
     override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
-        val rootView = View.inflate(activity, R.layout.email_dialog_fragment, null)
```

Comment:
The same XML file `email_dialog_fragment` was used for two alert dialogs namely `EmailLogoutDialog` and `WaitlistNotificationDialog` along with changing the text on runtime set from the XML file. 

The same behaviour is now achieved using the `.setMessage` from the **AlertDialog**

This may be out of the scope of the current issue, but I thought to anyway do the change. Please let me know if I need to undo these changes.